gadget/gadget.go

```diff
@@ -249,6 +249,12 @@ type DiskVolumeDeviceTraits struct {
 	// the volume that may be useful in identifying whether a disk matches a
 	// volume or not.
 	Structure []DiskStructureDeviceTraits `json:"structure"`
+
+	// EncryptedPartitions is the set of partitions that are encrypted on the
+	// volume - this should only ever be ubuntu-data or ubuntu-save for now, but
+	// the map will indicate the name of the structure as the key and the type
+	// of encryption (currently only "LUKS") as the value in the map.
+	EncryptedPartitions map[string]DiskEncryptionMethod `json:"enc-part-types"`
```

Comment:
shouldn't this be enc-structure? also would we possible need more information than the method? should the entries be maps

Reply:
I didn't change this because I'm still not sure if we need to do this

Reply:
ok, let's leave the value be just the types.  The JSON field name seems a bit out of place with the rest: "encrypted-structure",  or just "encrypted", "structure-encryption" ? yet something else?

Reply:
mmh actually I think you're right we should make it a map in case we want to add more values, we can just have one value for the method right now. Also sorry I didn't realize you were talking about the json key name, sure we can rename that

Reply:
ok, I did this now